### DISCUSSION TITLE
SMTP relay receiving the messages slow

### DISCUSSION THREAD

**Discussion by Praashanna** [postalserver/postal#3201] [Help with using Postal] [answered]
I was trying to set up an SMTP server and the mails seem to go out pretty fast. but, the receiving part (which I think is done by postal-smtp-1) is slow. It takes ages before the mail arrives at the postal before sending out to the external SMTPs.

Is there any way to scale smtp_server as the receiving seems slow? The mail goes out instantly once it is queued up, but receiving it from the original server is slow. Any fixes?

**Comment by willpower232** [comment]
Messages are sent by the workers so you may just need to scale them up as per other mentions in these discussions. If you look in the UI, do you have many messages queued?

**Comment by Praashanna** [comment]
No thats what im trying to say. They are queueing up slow.

**Reply by willpower232** [reply]
How are you giving the emails to Postal? Is that waiting for Postal to respond?

**Reply by Praashanna** [reply]
I'm not sure. The mails are handled by a forum software called XenForo. Is there any way I can test what is the bottleneck?

**Reply by Praashanna** [reply]
I can say that the logs for 1 email happen 1 second apart. So, thats pretty slow

**Reply by willpower232** [reply]
will XenForo tell you how long it takes to send each email?

**Reply by Praashanna** [reply]
it won't -> but from postal's admin : its around 60/min or 1 / sec

**Reply by Praashanna** [reply]
but, i can tell you this, it uses swiftmailer to send out messages.

**Reply by Praashanna** [reply]
and also, i can confirm it can go much faster -> I've tried another server with postal and it goes out faster. I'm not sure what is the problem here?

**Reply by Praashanna** [reply, marked as answer]
the problem was the server. the seller had DDoS protection ... now that we switched into different dedicated server, the speeds went up.
Thank you
